docs/dev/guidelines/client-tests.rst

```diff
@@ -314,3 +311,14 @@ Some guidelines:
   +--------------------------------------------------------+-----------------------------------------------------------------+
   | A spy should have been called once                     | :code:`expect(spy).toHaveBeenCalledTimes(1);`                   |
   +--------------------------------------------------------+-----------------------------------------------------------------+
+  | A spy should have been called with a value             | Always test the number of calls as well:                        |
+  |                                                        | :code:`expect(spy).toHaveBeenCalledTimes(1);`                   |
+  |                                                        | :code:`expect(spy).toHaveBeenCalledWith(value);`                |
+  |                                                        |                                                                 |
+  |                                                        | If you have multiple calls, you can verify the parameters       |
+  |                                                        | of each call separately:                                        |
+  |                                                        | :code:`expect(spy).toHaveBeenCalledTimes(3);`                   |
+  |                                                        | :code:`expect(spy).toHaveBeenNthCalledWith(0, value0);          |
+  |                                                        | :code:`expect(spy).toHaveBeenNthCalledWith(1, value1);          |
+  |                                                        | :code:`expect(spy).toHaveBeenNthCalledWith(2, value2);          |
```

Comment:
```suggestion
  |                                                        | :code:`expect(spy).toHaveBeenNthCalledWith(0, value0);`         |
  |                                                        | :code:`expect(spy).toHaveBeenNthCalledWith(1, value1);`         |
  |                                                        | :code:`expect(spy).toHaveBeenNthCalledWith(2, value2);`         |
```
The code blocks aren’t closed.

Additionally, the formatting could be improved by adding an empty line before each code line to display them align below each other instead of all on the same line.

Reply:
Replaced these multi-line additions with proper code blocks in 18a3eb4 👍

Reply:
Even nicer :+1: